tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/Main.java

```diff
@@ -64,10 +64,10 @@ private static void configureLoggerToJSONStdOut(Logger logger, String type, bool
         };
 
         if (json) {
-            GelfFormatter formatter = new GelfFormatter();
+            JsonFormatter formatter = new JsonFormatter();
             formatter.setTimestampPattern("yyyy-MM-dd HH:mm:ss,SSS");
-            formatter.setFields("Severity, Time, LoggerName");
-            formatter.setAdditionalFields("type=" + type);
+            formatter.setFields("severity, time, loggerName, thread");
+            formatter.setAdditionalFields("type=application,component=" + component);
```

Comment:
is it possible to make this typed:
formatter.setType("application");
formatter.setComponent(component);